### ISSUE TITLE
OMEdit Installation path issue on Windows Server 2012R2

### ISSUE THREAD

**Issue by 5upps** [OpenModelica/OpenModelica#9516]
### Description

On OS Windows Server 2012R2 I get this previous known issue where OMEdit can't run simulations when it's installed in a folder with a space in it like "C:\Program FIles". I then get an error saying "C:\Program is not a valid command".

I don't get this issue on OS Windows Server 2022 when installed in "C:\Program Files" or if I manually install the application on Windows Server 2012R2 in a folder that does not have a space in it's name.
This makes it difficult to distribute the application to multiple computers since it automatically selects the folder "C:\Program Files".

The fix to the issue came here: https://trac.openmodelica.org/OpenModelica/ticket/4504#no1
Is the fix not applicable on Windows Server 2012R2 or is there some additional file needed?

### Steps to Reproduce

Install OpenModelica 1.18.1-64bit on OS Windows Server 2012R2 in folder "C:\Program Files".

### Version and OS

 - OpenModelica Version: OpenModelica 1.18.1-64bit
 - OS: Windows Server 2012R2

**Comment by adrpo** [comment]
The fix requires support for short file names to be enabled:
https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/ff621566(v=ws.11)

`fsutil.exe behavior set disable8dot3 0`

**Comment by casella** [comment]
@5upps please reopen if you still have trouble with that.

**Comment by 5upps** [comment]
I ran `fsutil.exe behavior set disable8dot3 0` in cmd as admin. I rebooted the server and reinstalled OpenModelica. However the issue remains.

After some troubleshooting it seems that you also need to set a short name for the "C:\Programs Files" folder since previous command only enables this for new files.

You do this by booting into safe mode and then run command: `fsutil file setshortname "C:\Program Files" PROGRA~1`
If you don't do this in Safe mode you might be promted with "Access denied" since the OS uses the folder "C:\Program Files".

After this additional change OMEdit works when being installed in folder "C:\Program Files".
